src/pages/index.js

```diff
@@ -7,55 +7,56 @@ import UserInfo from "../components/UserInfo.js";
 import { initialCards } from "../utils/constants.js";
 import { config } from "../utils/constants.js";
 import ".//index.css";
-
-const cardData = {
-  name: "Zion Park",
-  link: "https://images.unsplash.com/photo-1587141105209-2a7ee89bce02?q=80&w=2987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
-};
-
-// handle image preview
-
-const handleImageClick = (cardData) => {
-  console.log(cardData);
-  popupWithImage.open(cardData);
-};
-
-// get all form elements
-const forms = document.querySelectorAll(config.formSelector);
-
-// initialize FormValidator for each form
-forms.forEach((form) => {
-  const formValidator = new FormValidator(config, form);
-  formValidator.enableValidation();
-});
+import Api from "../components/Api.js";
 
 const editButton = document.querySelector(".profile__edit-button");
-const editProfileModal = document.querySelector("#edit-modal");
-const profileTitle = document.querySelector(".profile__title");
-const profileSubtitle = document.querySelector(".profile__subtitle");
+
 const inputTitle = document.querySelector(".modal__input-title");
 const inputSubtitle = document.querySelector(".modal__input-subtitle");
-const profileCloseButton = document.querySelector(".modal__close-button");
-const profileEditForm = editProfileModal.querySelector(".modal__container");
-const cardList = document.querySelector(".cards__list");
-const cardTemplate = document.querySelector("#card-template");
 const addButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#modal-add-card");
-const closeAddFormButton = addCardModal.querySelector(".modal__close-button");
 const addCardForm = addCardModal.querySelector("#add-card-form");
-const cardName = addCardModal.querySelector(".modal__input-name");
-const cardLink = addCardModal.querySelector(".modal__input-link");
-const previewImageModal = document.querySelector("#modal-preview");
-const closePreviewButton = previewImageModal.querySelector(
-  ".modal__close-button"
-);
-const previewImageElement = previewImageModal.querySelector(
-  ".modal__preview-image"
-);
-const previewImageElementName = previewImageModal.querySelector(
-  ".modal__image-title"
+const profileAvatarContainer = document.querySelector(
+  ".profile__image-overlay"
 );
 
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6bc22e8b-f205-4c04-b4ee-f39ef81e1654",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getUserInfo()
+  .then((data) => {
+    userInfo.setUserInfo(data);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+
+  const handleAvatarSubmit = ({avatar}) => {
+    console.log(avatar);
+    editAvatarPopup.renderLoading(true);
+    api.updateAvatar(avatar)
+    .then((data) => {
+      userInfo.setUserInfo(data);
+      editAvatarPopup.close();
+      editAvatarPopup._form.reset();
```

Comment:
`_form`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.